docs/modules/tls.md

```diff
@@ -321,6 +321,55 @@ The alternative approach is easier for development, and that is to supply the PE
 will store the certificate into the flash chip and turn on verification for that certificate. Subsequent boots of the nodemcu can then
 use `tls.cert.verify(true)` and use the stored certificate.
 
+## tls.cert.auth()
+
+Controls the certificate verification process when the Nodemcu authenticates against the client like when receiving a secure connection.
+
+#### Syntax
+`tls.cert.auth(enable)`
+
+`tls.cert.auth(pemdata[, pemdata])`
+
+#### Parameters
+- `enable` A boolean which indicates whether verification should be enabled or not. The default at boot is `false`.
+- `pemdata` A string containing the CA certificate to use for verification. There can be several of these.
```

Comment:
This is not intended to be a CA certificate, but rather a certificate that corresponds to the client's private key, the other `pemdata` field.  There cannot, I think, be several of these.

Reply:
The examples given by @dtran123 suggest otherwise and the for-loop at https://github.com/nodemcu/nodemcu-firmware/blob/9080f49df6979217e512f75d0f080a5c3520ee12/app/modules/tls.c#L497 backs this up, no?

Reply:
Oh, no, right, of course the client might need to send a chain.  I retract my comment; thank you for looking.